pkg/kudoctl/packages/verifier/template/verify_render.go

```diff
@@ -50,16 +52,32 @@ func templateCompilable(pf *packages.Files) verifier.Result {
 			res.AddErrors(err.Error()) // err already mentions template name
 		}
 
-		// Try to parse rendered template as valid Kubernetes objects
-		_, err = renderer.YamlToObject(s)
-		if err != nil {
-			res.AddErrors(fmt.Sprintf("parsing rendered YAML from %s failed: %v", k, err))
+		// parameter file for KudoOperator task does not have to parse to kubernetes object
+		if !isParameterFile(k, pf) {
```

Comment:
not a big deal... but I'm wondering if we have have a 
`if isParameterFile()` that continues and reduce the indent on the error handling
From a Go perspective, it feels like a lot of nesting... reasonable to keep as well